wordpress-mobile/WordPress-Android
# Issue/4571 simperium lag on comment pn open show pn payload

Fixes #4571 

When simperium lags in syncing and we can't find the corresponding Note in the local bucket through its Note id,  this solution implements reading the Note from the Push Notification payload to be able to show it to the user.

To test: 

1. comment out the contents of method `startBuckets()`  in file `SimperiumUtils.java`. This will prevent Simperium from actually trying to sync buckets, so we make sure we are not getting info from Simperium as per the purpose of this test.
2. generate a comment on a blog you're an admin of (make the comment with another test user).
3. you should receive a push notification  (which internally carries the full not payload)
4. tap on such notification
5. the app will build the Note out of the PN payload and show the comment contents to the user.
6. also, the user should be able to moderate the comment accordingly

Needs review: @daniloercoli
